package.json

```diff
@@ -1,10 +1,19 @@
 {
+  "name": "career-motivation",
+  "scripts": {
+    "start": "NODE_ENV=production node src/server.js",
+    "mongo": "docker-compose up -d",
+    "dev": "NODE_ENV=development nodemon src/server.js"
```

Comment:
`npm start` probably won't be used locally but can be used in production.

`npm run mongo` is just a shortcut for the Docker Compose script.

`npm run dev` would be the script you run locally in development. I added `nodemon` to do auto-reloading when you make changes.